internal/measurex/easy.go

```diff
@@ -0,0 +1,266 @@
+package measurex
```

Comment:
We need to write tests for the whole of `measurex` before the next release. We need that because we want to ship a tested version of `websteps`. Hence I'm deferring writing tests for now.